packages/compass-utils/src/ipc.ts

```diff
@@ -5,21 +5,18 @@ type SerializedError = { $$error: Error & { statusCode?: number } };
 // We are serializing errors to get a better error shape on the other end, ipc
 // will only preserve message from the original error. See https://github.com/electron/electron/issues/24427
 function serializeErrorForIpc(err: any): SerializedError {
+  const props = Object.getOwnPropertyNames(err);
```

Comment:
Not all properties will be owned, some of them can come from the prototype chain and will not be returned here (this is what happens in case of abort error for example and this is why tests are failing)